AsyncDisplayKit/Layout/ASStackLayoutChild.h

```diff
@@ -0,0 +1,22 @@
+/*
+ *  Copyright (c) 2015-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+/**
+ Each child may override their parent stack's cross axis alignment.
+ @see ASStackLayoutAlignItems
+ */
+typedef NS_ENUM(NSUInteger, ASStackLayoutAlignSelf) {
```

Comment:
Yeah, reading the next file, it will actually make the stack system more clear to put this next to the AlignItems definition.  Improve the comment to explain that the property you set this on is defined on ASDisplayNode as well as ASLayoutSpec, so that the children of the stack layout (whether nodes or sublayouts) can also participate in how the stacking is performed.